src/cascadia/TerminalSettingsModel/KeyChordSerialization.cpp

```diff
@@ -5,95 +5,107 @@
 #include "KeyChordSerialization.h"
 #include "KeyChordSerialization.g.cpp"
 
+#include <til/static_map.h>
+
 using namespace winrt::Microsoft::Terminal::Control;
 using namespace winrt::Microsoft::Terminal::Settings::Model::implementation;
 using namespace Microsoft::Terminal::Settings::Model::JsonUtils;
 
-static constexpr std::wstring_view CTRL_KEY{ L"ctrl" };
-static constexpr std::wstring_view SHIFT_KEY{ L"shift" };
-static constexpr std::wstring_view ALT_KEY{ L"alt" };
-static constexpr std::wstring_view WIN_KEY{ L"win" };
+constexpr std::wstring_view CTRL_KEY{ L"ctrl" };
+constexpr std::wstring_view SHIFT_KEY{ L"shift" };
+constexpr std::wstring_view ALT_KEY{ L"alt" };
+constexpr std::wstring_view WIN_KEY{ L"win" };

```

Comment:
Because `constexpr` doesn't make any sense whatsoever. If you put a `constexpr` in global scope it's behaving as if it was static automatically. I can add `static` for consistency.